module/src/components/rangeInput/rangeInput.basic.scss

```diff
@@ -5,76 +5,75 @@ $range-input-track-height: 6px;
   display: flex;
   align-items: center;
 
-  .arm-range-input-track {
-    position: relative;
-    flex-grow: 1;
-    height: $input-height;
+  @include space-between-children-horizontal($spacing-xsmall);
 
-    input {
-      opacity: 0;
-      @include absolute-fill;
-      background-color: transparent;
-      border: none;
-      cursor: pointer;
+  &[data-disabled='true'] {
+    opacity: 0.5;
+  }
+}
 
-      @include range-input-thumb {
-        opacity: 0;
-        background-color: transparent;
-        appearance: none;
-      }
+.arm-range-input-track {
+  position: relative;
+  flex-grow: 1;
+  height: $input-height;
+}
 
-      @include range-input-track {
-        opacity: 0;
-        background-color: transparent;
-        appearance: none;
-      }
+.arm-range-input-input {
+  opacity: 0;
+  @include absolute-fill;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
 
-      &:active {
-        ~ .arm-range-input-handle {
-          transform: translate(-50%, -50%) scale(1.1);
-        }
-      }
-    }
+  @include range-input-thumb {
+    opacity: 0;
+    background-color: transparent;
+    appearance: none;
+  }
 
-    .arm-range-input-track-inner {
-      height: $range-input-track-height;
-      position: absolute;
-      width: 100%;
-      top: 50%;
-      transform: translateY(-50%);
-      pointer-events: none;
-      background-color: $gray-light;
-      overflow: hidden;
+  @include range-input-track {
+    opacity: 0;
+    background-color: transparent;
+    appearance: none;
+  }
 
-      &::after {
-        @include init-pseudo;
-        position: absolute;
-        left: 0;
-        top: 0;
-        width: var(--arm-range-input-percent);
-        height: 100%;
-        background-color: $brand-primary;
-      }
+  &:active {
+    ~ .arm-range-input-handle {
+      transform: translate(-50%, -50%) scale(1.1);
     }
+  }
+}
 
-    .arm-range-input-handle {
-      position: absolute;
-      left: var(--arm-range-input-percent);
-      transform: translate(-50%, -50%);
-      transition: transform $transition-quick;
-      @include circle(0.7rem);
-      top: 50%;
-      pointer-events: none;
+.arm-range-input-track-inner {
+  height: $range-input-track-height;
+  position: absolute;
+  width: 100%;
+  top: 50%;
+  transform: translateY(-50%);
+  pointer-events: none;
+  background-color: $gray-light;
+  overflow: hidden;
 
-      .arm-icon {
-        @include absolute-centre-self;
-      }
-    }
+  &::after {
+    @include init-pseudo;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: var(--arm-range-input-percent);
```

Comment:
I thought we weren't using CSS variables?

Reply:
My understanding is that we're not creating them for our general use variables, but we are using them when you need to set a property directly through JSX and the only alternative would be black-boxing it inside the JS with:
```
style={{ width: `${percent}%` }}
```